documentation/analytics/development.md

```diff
@@ -75,14 +68,7 @@ After choosing your approach, following the corresponding setup instructions:
 **Steps**
 
 1. Set up the project: `make setup` -- This will install the required packages and prompt you to authenticate with GitHub
-2. Set a GitHub Token in your terminal, via `export GH_TOKEN=...`. Acquiring the token is a multi-step process:
-  - Go to https://github.com/settings/tokens
-  - Generate a new token (classic)
-  - Give it the following scopes:
-    - repo
-    - read:org
-    - admin:public_key
-    - project
+2. Retrieve GH_TOKEN from [AWS](https://us-east-1.console.aws.amazon.com/systems-manager/parameters/%252Fanalytics%252Fgithub-token/description?region=us-east-1&tab=Table#list_parameter_filters=Name:Contains:analytics%2Fgithub-token)
```

Comment:
I'm torn because if we change this to "retrieve from the AWS console" then these instructions won't work as well for open source contributors. I would want @widal001 to weigh in on this one

Reply:
What if we make this an OR.
1. Acquire a token using one of the methods below
- Via AWS (Project team)
- Create your own in GitHub (Open Source)
- - Keep the original instructions here

Reply:
@mdragon We can certainly do that.